ui/src/app/utils/navigation.tsx

```diff
@@ -17,6 +17,7 @@ export const currentConceptSetStore = new BehaviorSubject<ConceptSet>(undefined)
 export const urlParamsStore = new BehaviorSubject<any>({});
 export const queryParamsStore = new BehaviorSubject<any>({});
 export const routeConfigDataStore = new BehaviorSubject<any>({});
+export const configDataStore = new BehaviorSubject<any>({});
```

Comment:
Should use a type - the above 3 are `any` because they originate from Angular and don't have a type (or it's complex).